src/pages/app/pages/PostInstall.tsx

```diff
@@ -150,13 +160,39 @@ const Steps = () => {
         onClick={onClick}
         title={t('postInstall_action_submit')}
       >
-        <div className="mx-auto h-popup w-popup overflow-hidden">
-          <iframe
-            loading="lazy"
-            src={popupUrl}
-            title="Connect your wallet"
-            className="h-popup w-popup border-none"
-          />
+        <div className="text-center mx-auto h-auto w-popup pt-4 pb-8 overflow-hidden">
```

Comment:
Consider extracting this step into a separate component, say, `StepConnectWallet`.